### PR TITLE
Add S01E01 placeholder

### PR DESCRIPTION
**Pull request by as2388**

### CODE REVIEW

**Review by lukechurch — APPROVED** [review]
Thanks @as2388

File: code_schemes/s01e01.json [hunk 1]
```diff
@@ -0,0 +1,162 @@
...
+      "NumericValue": -100060,
+      "StringValue": "participation_incentive",
+      "VisibleInCoda": true
+    },
+    {
+      "CodeID": "code-EC-3a704f5b",
+      "CodeType": "Meta",
+      "MetaCode": "exclusion_complaint",
+      "DisplayText": "meta: exclusion complaint",
+      "NumericValue": -100070,
```
Comment (IsaackMwenda) [inline code_schemes/s01e01.json:157]:
```suggestion
      "NumericValue": -100080,
```

Comment (IsaackMwenda) [reply code_schemes/s01e01.json:157]: They are sharing the same value with escalate

**Review by IsaackMwenda — APPROVED** [review]

File: code_schemes/s01e01.json (hunk 1 above)
Comment (as2388) [reply code_schemes/s01e01.json:157]: Ahhh good catch.

I actually need to fix the numeric value for escalate, on all of the existing schemes.
We should also add this check to the validators.

Comment (as2388) [reply code_schemes/s01e01.json:157]: corrected escalate instead

**Pull request merged**